vault/blog.2020.2020-08-24-a-hierarchy-first-approach-to-note-taking.md

```diff
@@ -267,7 +267,7 @@ A little over a year ago, I left my job at Amazon to work on a note-taking tool
 
 ## Dendron
 
-About a month ago, I launched the preview for [Dendron](https://dendron.so/). Dendron is the first-ever note-taking tool built from the ground up to support hierarchical note-taking. Dendron is open source, local first, Markdown-based, and runs as a collection of extension on top of VSCode.
+About a month ago, I launched the preview for [Dendron](https://link.dendron.so/home). Dendron is the first-ever note-taking tool built from the ground up to support hierarchical note-taking. Dendron is open source, local first, Markdown-based, and runs as a collection of extension on top of VSCode.
```

Comment:
we want to use switchly links on domain we don't control (eg. github, discord), but is there value in adding a switchly link in domains we do? we should already get referer values from google analytics